index.html

```diff
@@ -230,5 +230,38 @@ <h3 class="education-title">Skills</h3>
         </div>
         </section>
     </section>
+
+
+        <!-- Footer and contact form -->
+
+
+        <footersection class="form">
+            <div class="Contact-form-description">
+                <h1 class="Contact-form-description">
+                    I'm always interested in hearing
+                    about new projects, so if you'd
+                    like to chat please get in touch.
+                </h1>
+            </div>
+            <div id="form">
+                <form action="https://formspree.io/f/mpznrkvl" method="post">
+                    <input type="text" id="fullname" inputname="fullname" maxlength="30" placeholder="Full Name" required>
+                    <br>
+                    <input type="email" id="email" inputname="" placeholder="Email Address" required>
+                    <textarea name="message" id="textarea" placeholder="Enter Text Here" maxlength="500" required></textarea>
+                    <input type="submit" value="Get In Touch" id="btn-get-in-touch" class="submit-button">
+                </form>
+            </div>
+            <div id="social-media-links-2">
+                <ul id="media-links-2">
+                    <li><a href="https://github.com/187jjay187"><img src="Images/github.svg" alt="github"></a></li>
+                    <li><a href="https://linkedin.com"><img src="Images/Linkedin icon.svg" alt="Linkedin logo"></a></li>
+                    <li><a href="https://angel.co"><img src="Images/angellist.svg" alt="angellist"></a></li>
+                    <li><a href="https://twitter.com"><img src="Images/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="https://medium.com"><img src="Images/medium.svg" alt="medium"></a></li>
+                </ul>
+            </div>
```

Comment:
- [OPTIONAL] I suggest using `<footer id="social-media-links-2">` tag here instead of  `<div id="social-media-links-2">`, to follow best practices for semantic HTML5 tags 💯